config/samples/mellanox.com_v1alpha1_nicclusterpolicy.yaml

```diff
@@ -57,3 +57,40 @@ spec:
           }
         ]
       }
+  sriovDevicePlugin:
```

Comment:
@rollandf secondaryNetwork ipoib is also applicable on openshift

Reply:
Done